projects/ngx-extended-pdf-viewer/src/lib/toolbar/pdf-highlight-editor/pdf-highlight-editor.component.ts

```diff
@@ -38,4 +38,12 @@ export class PdfHighlightEditorComponent {
   public onClick(): void {
     document.getElementById('editorHighlight')?.click();
   }
+  //[FS] - 28-08-2024
+  @HostListener('document:keydown', ['$event'])
+  public handleOnClick(event: KeyboardEvent): void {
+    if (event.key === 'E' || event.key === 'e') {
+      event.preventDefault();
+      document.getElementById('editorHighlight')?.click();
```

Comment:
key check can be handled by the listener by using keydown.e. Also, this should call onClick() to avoid duplication incase the selector needs to change.